tests/Feature/Livewire/ServersTest.php

```diff
@@ -87,3 +87,47 @@
     Livewire::test(Servers::class, ['lazy' => false])
         ->assertSeeInOrder(['A Web', 'B Web', 'C Web']);
 });
+
+it('can ignore servers that have stopped reporting', function ($ignoreAfter, $see, $dontSee) {
+    Carbon::setTestNow(now()->startOfSecond());
+
+    $data = [
+        'memory_used' => 1234,
+        'memory_total' => 2468,
+        'cpu' => 99,
+        'storage' => [
+            ['directory' => '/', 'used' => 123, 'total' => 456],
+        ],
+    ];
+
+    Pulse::set('system', 'server-1', json_encode([
+        'name' => 'Server 1',
+        ...$data,
+    ]), now()->subSeconds(599));
+
+    Pulse::set('system', 'server-2', json_encode([
+        'name' => 'Server 2',
+        ...$data,
+    ]), now()->subSeconds(600));
+
+    Pulse::set('system', 'server-3', json_encode([
+        'name' => 'Server 3',
+        ...$data,
+    ]), now()->subSeconds(601));
+
+    Pulse::ingest();
+
+    Livewire::test(Servers::class, ['lazy' => false, 'ignoreAfter' => value($ignoreAfter)])
+        ->assertSeeInOrder($see)
+        ->assertDontSeeText($dontSee);
+})->with([
+    [null, ['Server 1', 'Server 2', 'Server 3'], []],
+    [588, [], ['Server 1', 'Server 2', 'Server 3']],
+    [599, ['Server 1'], ['Server 2', 'Server 3']],
+    [600, ['Server 1', 'Server 2'], ['Server 3']],
+    [601, ['Server 1', 'Server 2', 'Server 3'], []],
+    [now()->addSeconds(588), [], ['Server 1', 'Server 2', 'Server 3']],
+    [now()->addSeconds(599), ['Server 1'], ['Server 2', 'Server 3']],
+    [now()->addSeconds(600), ['Server 1', 'Server 2'], ['Server 3']],
+    [now()->addSeconds(601), ['Server 1', 'Server 2', 'Server 3'], []],
```

Comment:
These are wrapped in closures so they respect the `setTestNow` at the start of the test.